coding-challenges/README.md

```diff
@@ -46,7 +46,7 @@ All endpoints should work without any errors.
 Sorting algorithm should correctly sort the products in ascending order based on price.
 Test cases should cover basic scenarios and edge cases for the sorting algorithm.
 
-```
+```bash
```

Comment:
added bash on the code block because coding-challenge-1 had it, to keep things consistent

Reply:
If you look at the formatting of the README, it is somewhat messed up. You can see what it looks like under this link
https://github.com/berat-552/tech-vault/tree/feature-coding-challenge-2/coding-challenges